src/pages/concepts/cd/index.mdx

```diff
@@ -3,7 +3,7 @@ title: Continuous Delivery and Deployment
 description: Continuous Delivery and Deployment Concept
 ---
 
-<AnchorLinks>
+<AnchorLinks small>
```

Comment:
Oh there is a `small`?

Reply:
yeah it makes them smaller on the page which looks better